docs/map-layer-configuration.md

```diff
@@ -37,8 +36,8 @@ The following properties can be applied to all map layer types
 | selectable          | Boolean value, whether the features of the layer can be selected by click in order to display the attributes in a window | `"selectable": true` |
 | hoverable           | Boolean value, whether the features of the layer can be hovered in order to display an attribute (see `hoverAttribute`) in a tooltip  | `"hoverable": true` |
 | hoverAttribute      | Attribute to be shown if a feature of the layer is hovered. Only has an effect if `hoverable` is set to `true`  | `"hoverAttribute": "name"` |
-| style               | Object to define a rendering style for the features of the layer  | see [style](map-layer-configuration?id=style-for-vectorlayers) |
-| selectStyle         | The style for a selected feature | see [style](map-layer-configuration?id=style-for-vectorlayers) |
+| style               | Object to define a rendering style for the features of the layer  | see [style](map-layer-configuration.md?id=style-for-vectorlayers) |
```

Comment:
I tried to view the documentation on github via https://github.com/meggsimum/wegue/blob/master/docs/map-layer-configuration.md. You can see that clicking the links results in 404.

Reply:
yes that is true for viewing the pages on GitHub. But the docs are actually meant to be viewed via [docsify](https://docsify.js.org/#/) and are automatically deployed here: https://meggsimum.github.io/wegue/

The corresponding link to your example would be https://meggsimum.github.io/wegue/#/map-layer-configuration

In docsify it does not matter if the links end with `.md`. But it would be better to be consistent here. 
I would vote for not using `.md`, because it makes the url a bit cleaner.

Reply:
Ok got it. I will partially revert c5b90e44d0dd3aa98255739ea1fe920db2ae748d then.